transportable-udfs-test/transportable-udfs-test-trino/src/main/java/com/linkedin/transport/test/trino/TrinoTester.java

```diff
@@ -7,44 +7,62 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
-import io.trino.metadata.BoundSignature;
+import com.linkedin.transport.test.spi.TestCase;
+import com.linkedin.transport.test.spi.types.TestType;
+import io.trino.FeaturesConfig;
+import io.trino.Session;
+import io.trino.SessionTestUtils;
+import io.trino.metadata.InternalFunctionBundle;
+import io.trino.metadata.SqlFunction;
+import io.trino.spi.function.BoundSignature;
 import io.trino.metadata.FunctionBinding;
-import io.trino.metadata.FunctionId;
-import io.trino.operator.scalar.AbstractTestFunctions;
-import io.trino.spi.type.Type;
+import io.trino.spi.function.FunctionId;
 import com.linkedin.transport.api.StdFactory;
 import com.linkedin.transport.api.udf.StdUDF;
 import com.linkedin.transport.api.udf.TopLevelStdUDF;
 import com.linkedin.transport.trino.TrinoFactory;
 import com.linkedin.transport.test.spi.SqlFunctionCallGenerator;
 import com.linkedin.transport.test.spi.SqlStdTester;
 import com.linkedin.transport.test.spi.ToPlatformTestOutputConverter;
+import io.trino.spi.type.Type;
+import io.trino.sql.query.QueryAssertions;
+import io.trino.testing.LocalQueryRunner;
+import io.trino.type.InternalTypeManager;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
 import static io.trino.type.UnknownType.UNKNOWN;
+import static org.assertj.core.api.Assertions.*;
 
 
-public class TrinoTester extends AbstractTestFunctions implements SqlStdTester {
+public class TrinoTester implements SqlStdTester {
 
   private StdFactory _stdFactory;
   private SqlFunctionCallGenerator _sqlFunctionCallGenerator;
   private ToPlatformTestOutputConverter _toPlatformTestOutputConverter;
+  private Session _session;
+  private FeaturesConfig  _featuresConfig;
+  private LocalQueryRunner _runner;
+  private QueryAssertions _queryAssertions;
 
   public TrinoTester() {
     _stdFactory = null;
     _sqlFunctionCallGenerator = new TrinoSqlFunctionCallGenerator();
     _toPlatformTestOutputConverter = new ToTrinoTestOutputConverter();
+    _session = SessionTestUtils.TEST_SESSION;
+    _featuresConfig = new FeaturesConfig();
+    _runner = LocalQueryRunner.builder(_session).withFeaturesConfig(_featuresConfig).build();
+    _queryAssertions = new QueryAssertions(_runner);
   }
 
   @Override
   public void setup(
       Map<Class<? extends TopLevelStdUDF>, List<Class<? extends StdUDF>>> topLevelStdUDFClassesAndImplementations) {
     // Refresh Trino state during every setup call
-    initTestFunctions();
```

Comment:
could you clarify what was this init function doing previously and how come its not needed anymore?